spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -157,14 +160,46 @@ inline fun <reified T : Enum<T>> enumSetOf(vararg elems: T): EnumSet<T> =
  * license text is retrieved from that directory if and only if the license text is not known by ORT.
  */
 fun getLicenseText(id: String, handleExceptions: Boolean = false, customLicenseTextsDir: File? = null): String? =
+    getLicenseTextReader(id, handleExceptions, customLicenseTextsDir)?.read()
```

Comment:
When following my suggestion to return a function below, use `invoke()` instead of `read()` here.

Reply:
Done.